### PR TITLE
[test-only] expand createUser.feature

### PR DESCRIPTION
**Pull request by ScharfViktor**
related: #6606 

https://doc.owncloud.com/ocis/next/deployment/services/s-list/graph.html

tests for Env
- [x] GRAPH_USERNAME_MATCH
- [x] GRAPH_ASSIGN_DEFAULT_USER_ROLE
- [x] non-admin cannot change user role

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiGraph-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/23753/51/1) failed. Further test are cancelled...

File: tests/acceptance/features/apiGraph/assignRole.feature [hunk 1]
```diff
@@ -35,7 +35,7 @@ Feature: assign role
 
   Scenario Outline: assign role to the user with setting api and list role with graph api
     Given user "Alice" has been created with default attributes and without skeleton files
-    And the administrator has assigned the role "<userRole>" to user "Alice" using the Graph API
+    And the administrator has given "Alice" the role "<userRole>" using the settings api
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/assignRole.feature:38]: Why change this to setting api?

Comment (ScharfViktor) [reply tests/acceptance/features/apiGraph/assignRole.feature:38]: I read title: `Scenario Outline: assign role to the user with setting api and list role with graph api`

File: tests/acceptance/features/apiGraph/changeRole.feature [hunk 2]
```diff
@@ -46,3 +46,16 @@ Feature: change role
...
+    Given the administrator has assigned the role "<role>" to user "Alice" using the Graph API
+    And user "Brian" has been created with default attributes and without skeleton files
+    When user "Alice" tries to change the role of user "Alice" to role "Admin" using the Graph API
+    Then the HTTP status code should be "401"
+    And user "Brian" should have the role "User"
+    Examples:
+      | role     |
+      | Space Admin |
+      | User        |
+      | User Light  |
```
Comment (SwikritiT) [inline tests/acceptance/features/apiGraph/changeRole.feature:61]: The indent is little off
```suggestion
      | role        |
      | Space Admin |
      | User        |
      | User Light  |
```

**Review by SwikritiT — APPROVED** [review]

**Pull request merged**